platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -643,8 +809,8 @@ namespace platoon_strategic_ihp
     {   
```

Comment:
It's still really confusing when this method will be used and what the situation might be. This will be called when host is in the LEADER state, which I believe says it is either leading a platoon or driving solo.  Does the logic fully accommodate both of those situations?  The comments within the method seem to assume the host is a solo vehicle listening to a platoon leader.  Will a solo vehicle transmit INFO messages?  At a minimum, these things need to be explained in a comment block.